contrib/packaging/rpm/Makefile

```diff
@@ -1,18 +1,17 @@
 include ../../../Makefile.defs
 
-BUILDDIR := "$(CURDIR)/cilium-$(VERSION)"
+CWD := $(CURDIR)
+VERSION := $(shell echo $(VERSION)|cut -c 1-5 -)
```

Comment:
Builds based on my spec have following format: <version>-0.git<short_commit>. Short commit have 7 first characters - this is based on Fedora packaging guideline.